azure/Dockerfile

```diff
@@ -13,8 +13,7 @@ ARG FLYWAY_ARTIFACT_URL=https://repo1.maven.org/maven2/org/flywaydb/flyway-comma
 RUN apk add --no-cache --virtual .pipeline-deps readline linux-pam \
   && apk add bash sudo shadow openssl curl tar \
   && apk del .pipeline-deps
-RUN apk --no-cache add --update g++ python3 python3-dev py3-pip \
-    && pip3 install sqlfluff==1.2.1
+RUN 
```

Comment:
Empty `RUN` command?

Reply:
Thanks, missed that. Surprised it didn't break anything